### PR TITLE
Fix: two cases of PyXRF crashing

### PR DESCRIPTION
**Pull request by dmgav**
PyXRF was reported by to crash while plotting data for scan IDs 29929 and 29933. Two minor errors were corrected: rarely occurring division by 0 and plotting all-zeros image with imshow().

### CODE REVIEW

File: pyxrf/model/draw_image.py [hunk 1]
```diff
@@ -552,6 +558,10 @@ def _adjust_data_range__min_ratio(c_min, c_max, c_axis_range, *, min_ratio=0.01)
             else:
 
                 maxz = np.max(data_dict)
+                # Set some reasonable minimum range for the colorbar
+                #   Zeros or negative numbers will be shown in white
+                if maxz <= 1e-30:
```
Comment (mrakitin) [inline pyxrf/model/draw_image.py]: Thanks for the fix. That was actually my question.

File: pyxrf/model/draw_image.py [hunk 2]
```diff
@@ -574,6 +584,7 @@ def _adjust_data_range__min_ratio(c_min, c_max, c_axis_range, *, min_ratio=0.01)
                                          clim=(low_lim*maxz, maxz))
                     grid[i].set_ylim(yd_axis_min, yd_axis_max)
 
+
```
Comment (mrakitin) [inline pyxrf/model/draw_image.py]:
```suggestion
```

File: pyxrf/model/fit_spectrum.py [hunk 3]
```diff
@@ -1453,7 +1454,11 @@ def fit_per_line_nnls(row_num, data,
 
         result, res = nnls_fit(y, matv, weights=None)
         sst = np.sum((y-np.mean(y))**2)
-        r2_adjusted = 1 - res/(num_data-num_feature-1)/(sst/(num_data-1))
+        if not math.isclose(sst, 0, abs_tol=1e-10):
```
Comment (mrakitin) [inline pyxrf/model/fit_spectrum.py]: Why is it `1e-10` while in the change above it's `1e-20`? Should they be the same?

Comment (dmgav) [reply pyxrf/model/fit_spectrum.py]: Those changes are in different files and are not directly related. I just picked some small value. I don't know possible range for sst, but I think that it shouldn't go below 1e-10 for reasonable data input. I think we can change it to 1e-20. We don't plot 'sst' at all.

File: pyxrf/model/draw_image.py (hunk 1 above)
Comment (dmgav) [reply pyxrf/model/draw_image.py]: Yes, I noticed it by accident. Currently the colorbar axis labels are not displayed for log plots, so I didn't notice it. We are waiting for Matplotlib 3.2.

Comment (mrakitin) [reply pyxrf/model/draw_image.py]: OK.

File: pyxrf/model/fit_spectrum.py (hunk 3 above)
Comment (mrakitin) [reply pyxrf/model/fit_spectrum.py]: OK, thanks for the explanation.

**Pull request merged**